modules/fundamental/src/package/endpoints/base.rs

```diff
@@ -8,32 +8,32 @@ use trustify_common::model::Paginated;
     context_path= "/api",
     tag = "package",
     params(
-        ("uuid" = String, Path, description = "opaque UUID identifier for a base package")
+        ("id" = String, Path, description = "opaque identifier for a base PURL")
     ),
     responses(
-        (status = 200, description = "Details for the versionless base package", body = PackageDetails),
+        (status = 200, description = "Details for the versionless base PURL", body = PackageDetails),
     ),
 )]
-#[get("/v1/package/base/{uuid}")]
+#[get("/v1/package/by-purl/base/{id}")]
 pub async fn get(
     service: web::Data<PackageService>,
```

Comment:
Do we want to rename the service to PurlService?

Reply:
If we do, I'd argue the api should become `#[get("/v1/purl/base/{id}")]`

Reply:
Maybe not the service, but the functions and types that deal with PURLs. The main namespace/module still is `package`, which is ok.